Assets/Scripts/Jump.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Jump : MonoBehaviour {
+	
+	public float jumpSpeed = 8.0f;
+
+	private Vector3 moveDirection = Vector3.zero;
+	private bool grounded = true;
+	private InputController inputController;
+	private int playerNum;
+	private string jumpAxis;
+	// Use this for initialization
+	void Start () {
+		inputController = GameObject.FindWithTag ("GameController").GetComponent<InputController>();
+		string parentTag = this.transform.parent.tag;
+		string lastChar = parentTag.Substring (parentTag.Length - 1);
+		int.TryParse (lastChar, out playerNum);
```

Comment:
`lastChar` isn't descriptive enough to help me understand what it does. Would you give this a more informative variable name, please? I understand that it doesn't help that I don't know what `TryParse` is off-the-top-of-my-head.

Reply:
It gets the lastCharacter of the string

Reply:
If you want to rename it, I will do so when I work on 2 players since it relates to that and we use this variable name throughout the project if that's cool